### PR TITLE
GH-9679 ordering plugins in alphabetical order

### PR DESCRIPTION
**Pull request by s4kh**
#### Summary
Ordering plugins in alphabetical order (System console)

#### Ticket Link
https://github.com/mattermost/mattermost-server/issues/9679

#### Checklist
- [x] Ran `make check-style` to check for style errors (required for all pull requests)
- [x] Ran `make test` to ensure unit and component tests passed
- [x] Has UI changes

### CODE REVIEW

**Comment by jasonblais** [comment]
@cpanato Would you be able to help enable plugin uploads on the above test server?

**Comment by cpanato** [comment]
@jasonblais done, also I've enable the upload plugins by default for the test spinmints

File: components/admin_console/admin_sidebar/admin_sidebar.jsx [hunk 1]
```diff
@@ -343,7 +343,9 @@ export default class AdminSidebar extends React.Component {
 
         const customPlugins = [];
         if (this.props.config.PluginSettings.Enable) {
-            Object.values(this.props.plugins).forEach((p) => {
+            Object.values(this.props.plugins).sort((a, b) => {
+                return (a.name > b.name) ? 1 : -1;
```
Comment (lieut-data) [inline components/admin_console/admin_sidebar/admin_sidebar.jsx]: Minor issue, but this sort comparison isn't stable in the presence of duplicate `name` fields. This itself is relatively unlikely, but it should be easy to fix by comparing the `name` and then `id` with something like:

```js
Object.values(this.props.plugins).sort((a, b) => {
    const nameCompare = a.name.localeCompare(b.name);
    if (nameCompare !== 0) {
        return nameCompare;
    }

    return a.id.localeCompare(b.id);
});
```

**Comment by jasonblais** [comment]
Hey @s4kh - looks like your rebase resulted in extra commits for the PR.

Take a look at the answer to this [Stack Overflow question](http://stackoverflow.com/questions/17182624/contributing-to-project-on-github-how-to-rebase-my-pull-request-on-top-of-mast) Also here's the [Git page](https://git-scm.com/book/en/v2/Git-Branching-Rebasing) on rebasing.

Should help you rebase to fix the conflicts, but let us know if we can help with the rebasing! :)

**Comment by cpanato** [comment]
thanks @s4kh for this PR!

**Pull request merged**